public/css/style.css

```diff
@@ -0,0 +1,82 @@
+/* Great source: https://css-tricks.com/hexagons-and-beyond-flexible-responsive-grid-patterns-sans-media-queries/ */
+
+.gameboard {
+    display: grid;
+    place-self: center;
+    justify-content: center;
+    margin: 20px;
+    padding: 20px;
+    background: #333;
+}
+
+.hex-board-frame {
+    display: flex;
+    flex-direction: column;
```

Comment:
I don't think it's used anywhere because children are absolutely positioned here !

Reply:
indeed, it's a leftover from a previous attempt, thanks for pointing this out